go.mod

```diff
@@ -2,4 +2,23 @@ module github.com/envoyproxy/envoy
 
 go 1.20
 
-require google.golang.org/protobuf v1.32.0
+require (
+	github.com/census-instrumentation/opencensus-proto v0.4.1
+	github.com/cncf/xds/go v0.0.0-20231128003011-0fa0005c9caa
+	github.com/envoyproxy/go-control-plane v0.12.0
+	github.com/envoyproxy/protoc-gen-validate v1.0.2
+	github.com/golang/protobuf v1.5.3
+	github.com/prometheus/client_model v0.5.0
+	go.opentelemetry.io/proto/otlp v1.0.0
+	google.golang.org/genproto/googleapis/api v0.0.0-20240102182953-50ed04b92917
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20240102182953-50ed04b92917
+	google.golang.org/grpc v1.60.1
```

Comment:
Hello @phlax , these requires looks a bit weird. this is what I got, after `go mod tidy`:
```
module github.com/envoyproxy/envoy

go 1.20

require google.golang.org/protobuf v1.32.0

require github.com/google/go-cmp v0.5.9 // indirect
```

are there some untracked go source files in your side? i.e. some go files for local testing under the repo dir.
`go mod tidy` will generate requires for them too.

Reply:
unlikely

ftr this is what i ran

```console
docker run --rm -it -v $PWD:/workspace --workdir /workspace golang:1.20 go mod tidy
docker run --rm -it -v $PWD:/workspace --workdir /workspace/examples/golang-network/simple golang:1.20 go mod tidy
docker run --rm -it -v $PWD:/workspace --workdir /workspace/examples/golang-http/simple golang:1.20 go mod tidy
```

Reply:
if the deps generated are not correct - then lets update - as long as it actually works

Reply:
okay, created a PR: https://github.com/envoyproxy/envoy/pull/31640
it's regenerated by: `docker run --rm -it -v $PWD:/workspace --workdir /workspace golang:1.20 go mod tidy`